src/openrct2/drawing/LightFX.cpp

```diff
@@ -694,7 +696,7 @@ namespace OpenRCT2::Drawing::LightFx
         Add3DLight({ x, y, offsetZ }, lightType);
     }
 
-    uint32_t GetLightPolution()
+    static uint32_t GetLightPolution()
```

Comment:
Just noticed this has a typo. Correct would be `Pollution`, with an extra 'l'. Only worth addressing if it's little trouble.